crates/gpio/README.md

```diff
@@ -0,0 +1,17 @@
+# GPIO support
+
+This crate provides the ability for SpiderLighting applications to interact with GPIO pins.
+
+Pins are named in the slightfile and passed a configuration string with the following slash-separated parameters:
+
+- Pin number
+- Pin mode
+  - `input`
+  - `output`
+  - `pwm_output`
+- Optional mode-specific configuration
+  - For input: `pullup` or `pulldown`
+  - For output: initially set to `low` or `high`
+  - For PWM output: the PWM period in microseconds, if supported by the implementation
+
+See the [example application](../../examples/gpio-demo/).
```

Comment:
It's definitely not required to close this PR, but it might be helpful to add some comments to aid someone getting started w/ this capability – I have a Raspberry Pi at home like the one in your video, and would love to try using this (: